tap_github/repository_streams.py

```diff
@@ -15,6 +15,9 @@ class RepositoryStream(GitHubRestStream):
     MAX_RESULTS_LIMIT = 1000
 
     name = "repositories"
+    # updated_at will be updated any time the repository object is updated,
+    # e.g. when the description or the primary language of the repository is updated.
+    replication_key = "updated_at"
```

Comment:
The other option would be to used "pushed_at" instead

Reply:
or not use any replication_key for repo and set all ignore_parent_replication_key to True for sub streams?